common/religion/religions/water_group.txt

```diff
@@ -24,6 +24,20 @@
 	doctrine = doctrine_deviancy_accepted
 	doctrine = doctrine_witchcraft_accepted
 
+	#Magic
+	doctrine = doctrine_light_magic_ignored
+	doctrine = doctrine_shadow_magic_accepted
+	doctrine = doctrine_disorder_magic_crime
+	doctrine = doctrine_order_magic_ignored
+	doctrine = doctrine_life_magic_accepted
```

Comment:
Huh? Accepted? Why?

Reply:
Done?

Reply:
> Done?

Jinyu should have it for now. I would like it to be changed though https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/issues/158, but on a different branch.

Reply:
Done